### PR TITLE
Chaz ux

### PR DESCRIPTION
**Pull request by LilCthulu**
Removed all instances of category in project see commits for more detail.

### CODE REVIEW

**Comment by Anthony-Cortese** [comment]
Great job on this PR and removing the instances of category. It was awesome that you were able to find a new bug and fix that as well. My big question for those who are not familiar would be the why? The why this was needed to be in order to proceed. Other than that, this really looks great!

**Review by Anthony-Cortese — APPROVED** [review]
Great job on this PR and removing the instances of category. It was awesome that you were able to find a new bug and fix that as well. My big question for those who are not familiar would be the why? The why this was needed to be in order to proceed. Other than that, this really looks great!

File: src/components/forms/AddServiceTypeForm.js [hunk 1]
```diff
@@ -93,7 +80,7 @@ function AddServiceTypeForm({
             label="Program"
             rules={[
               {
-                required: true,
+                // required: true
```
Comment (santorap) [inline src/components/forms/AddServiceTypeForm.js:83]: Thanks for the additional information about why you had to comment this out. Sounds like there may be a mismatch in naming between the old db schema and the new one that was pushed in Labs 37.

**Review by santorap — APPROVED** [review]
Thanks for taking care of removing the legacy category code. These changes look good to me, though there are still some issues with the form not posting properly to the back end.

**Review by ashtilawat23 — APPROVED** [review]

**Pull request merged**